### DISCUSSION TITLE
Topic for stage channel

### DISCUSSION THREAD

**Discussion by vitiacat** [Rapptz/discord.py#7221] [Q&A] [answered]
Hello! How to set a topic for stage channel?

**Comment by oliver-ni** [comment, marked as answer]
Believe it's only available in master aka 2.0 beta. It's [`StageChannel.create_instance`](https://discordpy.readthedocs.io/en/master/api.html#discord.StageChannel.create_instance) if the stage isn't running at the moment. If it already is, you'd want [`StageInstance.edit`](https://discordpy.readthedocs.io/en/master/api.html#discord.StageInstance.edit).

**Reply by vitiacat** [reply]
Thanks!
